format/FlightSql.proto

```diff
@@ -760,6 +772,18 @@ enum SqlInfo {
   SQL_STORED_FUNCTIONS_USING_CALL_SYNTAX_SUPPORTED = 576;
 }
 
+// The level of support for Flight SQL transaction RPCs.
+enum FlightSqlTransactionSupport {
+  // Unknown/not indicated
+  FLIGHT_SQL_TRANSACTION_SUPPORT_UNKNOWN = 0;
+  // No supprot
+  FLIGHT_SQL_TRANSACTION_SUPPORT_NONE = 1;
+  // Transactions, but not savepoints
```

Comment:
what is the difference between transcations and save points?  Are there docs someplace?

Reply:
The terminology is taken from JDBC and some databases, e.g. see [Postgres](https://www.postgresql.org/docs/current/sql-savepoint.html), [JDBC](https://docs.oracle.com/javase/8/docs/api/java/sql/Connection.html#setSavepoint-java.lang.String-). I will add some more language so this is reasonably self-contained.